src/bacnet/basic/object/csv.c

```diff
@@ -90,17 +102,21 @@ void CharacterString_Value_Property_Lists(
  * Initialize the character string values.
  */
 void CharacterString_Value_Init(void)
-{
+{       
     unsigned i;
 
     /* initialize all Present Values */
     for (i = 0; i < MAX_CHARACTERSTRING_VALUES; i++) {
-        snprintf(&Object_Name[i][0], sizeof(Object_Name[i]),
+        memset(&CSV_Descr[i], 0x00, sizeof(CHARACTERSTRING_VALUE_DESCR));
+        memset(&Present_Value[i], 0x00, sizeof(MAX_CHARACTERSTRING_VALUES)); 
+        snprintf(CSV_Descr[i].Name, sizeof(CSV_Descr[i].Name),
             "CHARACTER STRING VALUE %u", i + 1);
-        snprintf(&Object_Description[i][0], sizeof(Object_Description[i]),
```

Comment:
has object description become unused? if so let's remove it....

Reply:
This is not in my repo, which is confusing